File: CS/DXApplication191/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.LookAndFeel;
using DevExpress.Skins;
using DevExpress.Utils;
using DevExpress.Utils.Drawing;
using DXApplication191.Properties;

namespace DXApplication191 {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        SkinButtonPaintHelper helper;
        public Form1() {
            InitializeComponent();
            helper = new SkinButtonPaintHelper() {
                ButtonImage = Resources.open_32x32,
                ButtonRectangle = new Rectangle(5, 5, 200, 80),
                ButtonText = "Manually painted"
            };
        }
        protected override void OnPaint(PaintEventArgs e) {
            base.OnPaint(e);
            helper.DrawButton(e, ObjectState.Normal);
        }
    }

}

File: VB/DXApplication191/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.LookAndFeel
Imports DevExpress.Skins
Imports DevExpress.Utils
Imports DevExpress.Utils.Drawing

Namespace DXApplication191
    Partial Public Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Private _helper As SkinButtonPaintHelper
        Public Sub New()
            InitializeComponent()
            _helper = New SkinButtonPaintHelper() With {.ButtonImage = My.Resources.open_32x32, .ButtonRectangle = New Rectangle(5, 5, 200, 80), .ButtonText = "Manually painted"}
        End Sub
        Protected Overrides Sub OnPaint(ByVal e As PaintEventArgs)
            MyBase.OnPaint(e)
            _helper.DrawButton(e, ObjectState.Normal)
        End Sub
    End Class

End Namespace
